.github/workflows/_reusable_app_release.yml

```diff
@@ -162,6 +161,8 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-1
+      - name: Trim unused localizations from string catalogs
+        run: find . -name "*.xcstrings" | xargs xcrun swift run --package-path ./scripts TrimStringCatalogs
```

Comment:
so we trim before submitting?
what about trimming when we do update the strings, so during the crowdin job?

Reply:
I considered that, but the update action runs on ubuntu which probably (haven't verified actually) doesn't have any swift runtime installed.

Reply:
I was able to achieve it now.